lib/components/channel_panel.dart

```diff
@@ -69,103 +59,27 @@ class _ChannelPanelWidgetState extends State<ChannelPanelWidget> {
-                        ),
-                        Expanded(
-                          child: Text("/${first.displayName}",
-                              softWrap: false, overflow: TextOverflow.fade),
-                        ),
-                      ]),
-                  onSelected: (value) async {
-                    if (value.isAdd) {
-                      // show the search dialog.
-                      await showDialog(
-                        context: context,
-                        builder: (context) =>
-                            ChannelSearchDialog(onSelect: (channel) {
-                          channelsModel.subscribedChannels = {channel};
-                          Navigator.pop(context);
-                        }),
-                      );
-                    } else {
-                      channelsModel.subscribedChannels = {value.channel!};
-                    }
-                  },
-                  itemBuilder: (context) {
-                    return [
-                      ...channelsModel.availableChannels.map((channel) {
-                        return PopupMenuItem(
-                            value: _ChannelPickerValue(channel: channel),
-                            child: ListTile(
-                              title: Text(channel.displayName),
-                            ));
-                      }),
-                      const PopupMenuItem(
-                          value: _ChannelPickerValue(isAdd: true),
-                          child: ListTile(
-                              title: Text("Find a channel"),
-                              leading: Icon(Icons.add)))
-                    ];
-                  }),
-            );
-          }),
-          Consumer<ChannelsModel>(builder: (context, channelsModel, child) {
-            if (channelsModel.subscribedChannels.isEmpty) {
-              return Container();
-            }
-            final first = channelsModel.subscribedChannels.first;
-            return StatisticsBarWidget(
-                provider: first.provider, channelId: first.channelId);
-          }),
-          Consumer<TtsModel>(builder: (context, ttsModel, child) {
-            return IconButton(
-                icon: Icon(ttsModel.enabled
-                    ? Icons.record_voice_over
-                    : Icons.voice_over_off),
-                tooltip: "Text to speech",
-                onPressed: () {
-                  ttsModel.enabled = !ttsModel.enabled;
-                });
-          }),
-        ]),
+        child: Consumer<LayoutModel>(builder: (context, layoutModel, child) {
+          if (layoutModel.locked) {
+            return Container();
+          }
+          return Center(
+            child: ClipRRect(
+              borderRadius: BorderRadius.circular(15.0),
+              child: Container(
+                width: 350,
+                height: 16,
+                color: Colors.blueAccent,
```

Comment:
I think we might want to use a color based on theme? How does the bar look on dark mode?

Reply:
Yeah I think the Android one looks better. This is a bit tricky because it's not often we have this type of content where we need a split in apps so there might not be many good examples.